src/Evolution/Systems/ScalarWave/Constraints.hpp

```diff
@@ -0,0 +1,109 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/ScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+
+/// \cond
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace ScalarWave {
+// @{
+/*!
+ * \brief Computes the scalar-wave one-index constraint.
```

Comment:
Nitpick: `Computes` -> `Compute` (we formulate `\brief` function documentation as actionable sentences)

same in the other functions

Reply:
done